src/tdigest.c

```diff
@@ -4,6 +4,12 @@
 #include <math.h>
 #include "tdigest.h"
 
+#ifndef TD_MALLOC_INCLUDE
+#define TD_MALLOC_INCLUDE "td_malloc.h"
+#endif
+
+#include TD_MALLOC_INCLUDE
```

Comment:
Why not simply include the`.h` file?

Reply:
@ashtul this will allow to specify a different allocator include file via define.

Reply:
Example of antirez rax: https://github.com/filipecosta90/rax/commit/ca11410a2791041aef01d2d83e1c40be8a33feea